parquet/src/main/java/org/apache/iceberg/parquet/ReadConf.java

```diff
@@ -85,6 +87,9 @@
     this.rowGroups = reader.getRowGroups();
     this.shouldSkip = new boolean[rowGroups.size()];
 
+    Map<Long, Long> offsetToStartRowPosMap = generateRowGroupsStartRowPos();
```

Comment:
How about naming this `offsetToStartPos` and similarly updating the method name? There's no need to include a type in the variable name, usually.

Reply:
OK, done.